addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10313,7 +10313,11 @@ msgctxt "#20244"
 msgid "Android Apps"
 msgstr ""
 
-#empty strings from id 20245 to 20249
+msgctxt "#20245"
```

Comment:
Hooking settings through vfs sounds really odd IMO. Why would that be
beneficial? To alter them you need to write and bow would you do that
through our GUI? Via file manager?

Would it be far better to hook it into our settings system. Even if it
means to extend it. So from a interaction standpoint os, kodi and in
extension add-on settings are separated but they share the same plumbing. I
haven't looked at the code but vfs just immediately sounds really odd :)